guides/concepts/fees.md

```diff
@@ -6,8 +6,8 @@ The Stellar network imposes small fees on transactions and minimum balances on a
 
 There are two special values used to calculate fees:
 
-1. The **base fee** (100 stroops) is used in transaction fees.
-2. The **base reserve** (10 XLM) is used in minimum account balances.
+1. The **base fee** (currently 100 stroops) is used in transaction fees.
+2. The **base reserve** (currently 10 XLM) is used in minimum account balances.
```

Comment:
Ha! This is what I’d originally wrote, then figured dropping the "currently" bit was just as clear and more concise, given the section on changes below. :P

I think it’s fine to add back in if you think it’s important to note here, though.